ui/ui-components/pages/group/[id]/edit.tsx

```diff
@@ -158,7 +158,7 @@ export default function Page(props) {
                   disabled={loading}
                   size="sm"
                   onClick={() => {
-                    handleDelete();
+                    handleDelete(process.env.GROUPAROO_UI_EDITION === "config");
```

Comment:
I _feel_ like just for consistency maybe both `handleDelete` functions (in group and in destination edit) should work the same way, e.g. both taking a `force` arg or both checking the env var within the function.

Reply:
Thanks for the thoughtful review, @pedroslopez!

This shape is slightly different because there is an enterprise workflow for groups in which you can force delete a group after deleting it. Therefore, there is a call above this one that says `handleDelete(true)`. So groups are going to differ from destinations in some way. This seemed like an easy path to get there.

What do you think?

Reply:
Ah, I see! In that case the destination's `handleDelete` could take an argument in the same way, but it's fine either way. I don't feel too strongly about it 🙂